### ISSUE TITLE
Unable to set LoadBalancerIp

### ISSUE THREAD

**Issue by sblaster** [emqx/emqx-rel#466]
#### Environment

- OS: K8S
- Erlang/OTP: 
- EMQ: 3.2.1
- Charts : 4.0.5

#### Description

I'm trying to deploy EMQX on a AKS K8S via Rancher. The idea is to use Azure LoadBalancer on a clustered EMQX. Everything is working great, except that the Public IP is renewed each deployement.

Indeed, in AKS, i have to add serivce.LoadBalancerIp to specify the Public IP already created in Azure. Is there a way to inject this param within the answer of the helm chart ?

 ```
 emqxConfig: 
    EMQX_CLUSTER__K8S__ADDRESS_TYPE: "Ip"
  service: 
    type: "LoadBalancer"
    loadBalancerIP: "104.*.*.*"

```

**Comment by zhanghongtong** [comment]
Sorry, our helm chart package is not friendly to LoadBalancer, we will improve this issue later, or you can also contribute PR to us

**Comment by zhanghongtong** [comment]
Hi, @sblaster 
This feature has been implemented in the v4.1-rc.2 version, welcome to use, I will turn off this issue first, if there are any problems, you can always reopen it
